src/components/avatar/Avatar.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import Icon, {TYPE as ICON_TYPE, COLOR as ICON_COLOR} from '../icons/Icon';
+import Icon, {TYPE as ICON_TYPE, COLOR as ICON_COLOR} from 'icons/Icon';
```

Comment:
@kdzwinel @vergilius @matikrk this way or from 'components/icons/Icon';
same question for 'DocsBlock' or 'docs/DocsBlock' ?